prometheus/prometheus_two.yml

```diff
@@ -16,13 +16,11 @@ scrape_configs:
     static_configs:
       - targets: ['minio:9000']
 
-  - job_name: 'thanos_sidecar_two'
-    scrape_interval: 100ms
+  - job_name: 'thanos-sidecar_two'
     static_configs:
       - targets: ['thanos_sidecar_two:10902']
 
-  - job_name: 'thanos_querier'
-    scrape_interval: 100ms
```

Comment:
Why are we selectively removing this? I think we can remove this for all of them and rely on the global `scrape_interval`, right?

Reply:
This could probably be a separate PR. I think global interval makes more sense. 100ms was killing the laptop I was testing on and seems pretty aggressive to me.

Reply:
Yep, I agree. I am going to merge this, and then let's work on a second PR to unify scrape intervals. Thanks for the work!